tests/sentry/api/endpoints/test_project_profiling_profile.py

```diff
@@ -12,3 +17,29 @@ def setUp(self):
     def test_feature_flag_disabled(self):
         response = self.get_response(self.project.organization.slug, self.project.id, str(uuid4()))
         assert response.status_code == 404
+
+
+class ProjectProfilingFunctionsEndpoint(APITestCase):
```

Comment:
Are there any test for successful responses?

Reply:
No, because the actual endpoint isn't implemented in sentry. This is just serving as a proxy to the profiling service.